### PR TITLE
AGP-28: Decentralizing aragonpm.eth

### PR DESCRIPTION
**Pull request by izqui**

### CODE REVIEW

File: AGPs/AGP-W.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+AGP: N/A
+Title: Decentralizing `aragonpm.eth`
```
Comment (john-light) [inline AGPs/AGP-W.md]:
```suggestion
Title: Decentralizing aragonpm.eth
```

File: AGPs/AGP-W.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+- Mainnet and rinkeby environments:
+    - Change the root authority of the aragonPM DAOs from the 'Aragon One cold wallet' to the Aragon Association multisig. A Voting app within the DAO will control creating new repos and publishing new versions to all the repos (see Appendix for the proposed DAO structure)
+    - Create a new aragonPM registry at `a1.aragonpm.eth` controlled by Aragon One, granting 'Aragon One cold wallet' the `CREATE_REPO_ROLE` permission in the registry and setting this address as the root authority of the DAO. This registry can be used by Aragon One to publish apps before releasing them in `aragonpm.eth`.
+- Staging: Aragon One will keep control of the current staging environment, renaming it to `a1-staging` (in both https://aragonpm.com and deployments documentation). The rationale being that other teams can very easily create their own staging environments if they need one (the deployment process is documented in [aragon/deployments](https://github.com/aragon/deployments)) or use the `open.aragonpm.eth` registry that allows anyone to create their own repos.
+
+## Appendix: `aragonpm.eth` DAO structure
```
Comment (john-light) [inline AGPs/AGP-W.md]:
```suggestion
## Appendix: `aragonpm.eth` DAO structure
```

File: AGPs/AGP-W.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+(*) denotes that this is a permission that will need to be modified or be created
+
+#### Initial token distribution
+
+APM token holders:
+
+- Aragon One: 10 APM
+- Aragon Network Security Partner (at the time of deployment): 5 APM
+
+The AA multisig has the ability to change the distribution of token holders by minting and burning tokens. These changes should generally be triggered by an AGP (p.e. a new Flock team that requests APM tokens as part of their proposal) but as per AGP-1, the Aragon Association would still be able to perform these changes unilaterally (p.e. a Flock team is no longer working for the Network).
```
Comment (john-light) [inline AGPs/AGP-W.md]:
```suggestion
The AA multisig has the ability to change the distribution of token holders by minting and burning tokens. These changes should generally be triggered by an AGP (e.g. a new Flock team that requests APM tokens as part of their proposal) but as per AGP-1, the Aragon Association would still be able to perform these changes unilaterally (e.g. a Flock team is no longer working for the Network).
```

File: AGPs/AGP-W.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
```
Comment (john-light) [inline AGPs/AGP-W.md]:
```suggestion
```

File: AGPs/AGP-W.md (hunk 2 above)
Comment (john-light) [reply AGPs/AGP-W.md]: For whatever reason the space in the original text wasn't rendering the Markdown properly. It looks fixed when I deleted and added a space here.

File: AGPs/AGP-W.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+|     APMRegistry    |    CREATE_REPOS    |     Voting (*)    | AA multisig (*) |
+| All Repo instances |   CREATE_VERSION   |     Voting (*)    | AA multisig (*) |
+|       Voting       |    CREATE_VOTES    | Token Manager (*) | AA multisig (*) |
+|       Voting       |   MODIFY_SUPPORT   |  AA multisig (*)  | AA multisig (*) |
+|       Voting       |    MODIFY_QUORUM   |  AA multisig (*)  | AA multisig (*) |
+|    Token Manager   |        MINT        |  AA multisig (*)  | AA multisig (*) |
+|    Token Manager   |        BURN        |  AA multisig (*)  | AA multisig (*) |
+
+- Mainnet AA multisig: 0xcafE1A77e84698c83CA8931F54A755176eF75f2C
+- Rinkeby AA multisig: TBD by Aragon Association
```
Comment (john-light) [inline AGPs/AGP-W.md]: From previous experience with the AGP-10 Community Funding DAO we may want to ensure that the address here is specified just so there is no misunderstanding later. Although the stakes are low as this is just a Rinkeby DAO, I thought it worth pointing out.

File: AGPs/AGP-W.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+It's been more than 5 months after releasing Aragon 0.6. During this time we have published 4 minor releases, all of which were properly scheduled and no emergency releases have been done. We consider that it is now safe and appropriate to transition control of the aragonPM registries from the Aragon One cold key to a more decentralized and future-proof publishing process that won't solely depend on Aragon One.
+
+## Description of desired Association policy change
+
+The following changes are proposed to the `aragonpm.eth` aragonPM DAOs:
+
+- Mainnet and rinkeby environments:
+    - Change the root authority of the aragonPM DAOs from the 'Aragon One cold wallet' to the Aragon Association multisig. A Voting app within the DAO will control creating new repos and publishing new versions to all the repos (see Appendix for the proposed DAO structure)
+    - Create a new aragonPM registry at `a1.aragonpm.eth` controlled by Aragon One, granting 'Aragon One cold wallet' the `CREATE_REPO_ROLE` permission in the registry and setting this address as the root authority of the DAO. This registry can be used by Aragon One to publish apps before releasing them in `aragonpm.eth`.
```
Comment (sohkai) [inline AGPs/AGP-W.md]: One potentially painful spot here is on how the CLI works with app names. Right now there's no easy way to modify the registry you want, without completely changing the app's name each time in its arapp.json.

Comment (0x6431346e) [reply AGPs/AGP-W.md]: Could you please expand on this? Isn't this as easy as adding a new environment? 🤔

Comment (sohkai) [reply AGPs/AGP-W.md]: Ah ha, yes, you're right! Forgot that the environments let you specify the full app name :).

File: AGPs/AGP-W.md (hunk 5 above)
Comment (izqui) [reply AGPs/AGP-W.md]: Thanks for pointing it out, given that this is just for Rinkeby I would leave it as it is now.

**Review by john-light — DISMISSED** [review]

**Comment by john-light** [comment]
Please let the AGP editors know when your proposal has been finalized and you would like to submit it for final review by the Aragon Association. The deadline to finalize and submit your proposal is 16:00 April 11th. Proposals not finalized at this time will not be submitted for review.

**Comment by izqui** [comment]
Proposal is final, can be submitted for review

**Review by john-light — APPROVED** [review]

**Review by john-light — APPROVED** [review]

**Comment by john-light** [comment]
A technical issue with GitHub is preventing this PR from being merged, but from an AGP Editorial perspective this AGP has been finalized and will be reviewed by the Aragon Association Board of Directors for ANV-02. We will try to resolve this technical issue as soon as possible.

**Comment by luisivan** [comment]
@izqui can you try signing the CLA now?

**Pull request merged**